### PR TITLE
Fix emulate-livesync-emulate

### PR DESCRIPTION
**Pull request by Fatme**
Fixes https://github.com/NativeScript/nativescript-cli/issues/1031

### CODE REVIEW

File: mobile/android/android-emulator-services.ts [hunk 1]
```diff
@@ -45,6 +45,21 @@ class AndroidEmulatorServices implements Mobile.IEmulatorPlatformServices {
 		this.adbFilePath = this.$staticConfig.getAdbFilePath().wait();
 	}
 
+	public getEmulatorId(): IFuture<string> {
+		return (() => {
+			let image = this.getEmulatorImage().wait();
+			let emulatorId: string = null;
+
+			if(image) {
```
Comment (teobugslayer) [inline mobile/android/android-emulator-services.ts]: Not a merge stopper, but this code could be simplified by checking for the error condition first:

```
if (!image) fail();
return this.startInstance().wait();
```

**Comment by teobugslayer** [comment]
:+1:

**Comment by justcodebuilduser** [comment]
:heart:

**Comment by justcodebuilduser** [comment]
:heart:

**Pull request merged**